src/main/java/bio/terra/cli/app/utils/VersionCheckUtils.java

```diff
@@ -5,55 +5,32 @@
 import bio.terra.cli.service.WorkspaceManagerService;
 import bio.terra.workspace.model.SystemVersion;
 import java.lang.module.ModuleDescriptor.Version;
-import java.time.Duration;
 import java.time.OffsetDateTime;
-import java.util.Optional;
 import javax.annotation.Nullable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class VersionCheckUtils {
   private static final Logger logger = LoggerFactory.getLogger(VersionCheckUtils.class);
-  private static final Duration VERSION_CHECK_INTERVAL = Duration.ofMinutes(30);
 
   /** Query Workspace Manager for the oldest supported */
   public static boolean isObsolete() {
-    if (checkIntervalElapsed()) {
```

Comment:
Based on the change, we should check the version once the command request the server, so the `checkIntervalElapsed` is not need. If we keep this, when the version is out of date. it will show nothing until 30 minutes later. Like
```
ginay-macbookpro:terra-cli ginay$ terra workspace list
ginay-macbookpro:terra-cli ginay$ 
```
This is also not clear for the reason why this is nothing print out, Maybe because of the server does not have workspace, or some error hit.